helm/chart-operator-chart/Chart.yaml

```diff
@@ -1,4 +1,4 @@
 apiVersion: v1
 description: A Helm chart for the chart-operator
 name: chart-operator-chart
-version: 0.3.2
+version: 0.4.0
```

Comment:
Bumping version to align with version bundle

Reply:
Is this also aligned with the indexblob?

Reply:
Or should it be? Maybe I am confused here.